### ISSUE TITLE
activation functions to the final results

### ISSUE THREAD

**Issue by trikim** [dangweili/pedestrian-attribute-recognition-pytorch#4]
Excuse me, have you tried some activation functions(sigmoid, softmax etc.) to the final score of resnet50? can these operations improve the performance of the model? I am now trying.

**Comment by dangweili** [comment]
Typically, there is a sigmoid layer after the score to transform the score into a probability for binary attribute classification. 
If you use the score, the threshold for the positive attribute is 0.
If you use a sigmoid layer after the score, the threshold for the positive attribute is 0.5.

For binary attribute classification, you can also try the softmax layer (2 outputs), which is the same as the sigmoid layer, and the main difference is one or two outputs.
